File: source/zombie/commands/serverCommands/ArgType.java
// Decompiled on Sat Jan 17 08:24:00 EST 2026 with Zomboid Decompiler v0.2.3 using Vineflower.
package zombie.commands.serverCommands;

public class ArgType {
    public static final String PlayerName = "(.+)";
    public static final String AnyText = "(.+)";
    public static final String Script = "([a-zA-Z0-9.-]*[a-zA-Z][a-zA-Z0-9_.-]*)";
    public static final String Coordinates = "(\\d+),(\\d+),(\\d+)";
    public static final String IP = "((?:\\d{1,3}\\.){3}\\d{1,3})";
    public static final String TrueFalse = "(-true|-false)";
    public static final String ItemName = "([a-zA-Z0-9.-]*[a-zA-Z][a-zA-Z0-9_.-]*)";
    public static final String Value = "(\\d+)";
}

File: DoNotUse/include/zombie/commands/serverCommands/ArgType.cpp
#pragma once
#include <string>
#include <vector>
#include <memory>
#include <unordered_map>
#include <unordered_set>
#include <cstdint>

namespace zombie {
namespace commands {
namespace serverCommands {

class ArgType {
public:
    static const std::string PlayerName = "(.+)";
    static const std::string AnyText = "(.+)";
    static const std::string Script = "([a-zA-Z0-9.-]*[a-zA-Z][a-zA-Z0-9_.-]*)";
    static const std::string Coordinates = "(\\d+),(\\d+),(\\d+)";
    static const std::string IP = "((?:\\d{1,3}\\.){3}\\d{1,3})";
    static const std::string TrueFalse = "(-true|-false)";
    static const std::string ItemName = "([a-zA-Z0-9.-]*[a-zA-Z][a-zA-Z0-9_.-]*)";
    static const std::string Value = "(\\d+)";
}
} // namespace serverCommands
} // namespace commands
} // namespace zombie
